packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4747,15 +4742,15 @@ class StatefulElement extends ComponentElement {
   void update(StatefulWidget newWidget) {
     super.update(newWidget);
     assert(widget == newWidget);
-    final StatefulWidget oldWidget = _state._widget;
+    final StatefulWidget oldWidget = _state!._widget!;
     // Notice that we mark ourselves as dirty before calling didUpdateWidget to
```

Comment:
please do me a favour and remove the "Notice that" here... :-)

Reply:
Done